python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -637,6 +638,16 @@ def test_namedargs_with_global_limit(self):
         df2 = self.spark.sql(sqlText, args={"val": 1})
         self.assert_eq(df.toPandas(), df2.toPandas())
 
+        self.assert_eq(df.first()[0], datetime.datetime(2022, 12, 25, 10, 30))
+        self.assert_eq(df.first().date, datetime.datetime(2022, 12, 25, 10, 30))
+        self.assert_eq(df.first()[1], 1)
+        self.assert_eq(df.first().val, 1)
+
+        self.assert_eq(df.head()[0], datetime.datetime(2022, 12, 25, 10, 30))
```

Comment:
isn't `df.head` the same as `df.first()`?

Reply:
It is. Removed redundancy.